React/Views/RCTViewManager.m

```diff
@@ -197,7 +197,7 @@ - (RCTShadowView *)shadowView
 RCT_REMAP_VIEW_PROPERTY(opacity, alphaValue, CGFloat)
 #endif // ]TODO(macOS GH#774)
 
-#if TARGET_OS_OSX // TODO(macOS GH#774)
+#if !TARGET_OS_OSX // TODO(macOS GH#774)
```

Comment:
how did this work before? and nothing failed?

Reply:
Yeah I had the same question, I didn't test on IOS when I made this change but I feel like I should have seen errors when I tested on macOS

Reply:
Perhaps a copy-paste error since you tested in FURN and not RN-Tester?